docs/MAINTENANCE.md

```diff
@@ -2,25 +2,27 @@
 
 ## What is it
 
-There are four satis servers which are built from four config files (in ./satis-config).
-These can be used in the "repositories" section of composer.json.
+There are four satis servers which are built from four config files (in 
+`./satis-config`).
+These can be used in the "repositories" section of `composer.json`.
 
 |   |   |   |
 | --- | --- | --- |
-| `STABLE` | govcms-stable.json | https://satis.govcms.gov.au/ |
-| `MASTER` | govcms-master.json | https://satis.govcms.gov.au/master |
-| `DEVELOP` | govcms-develop.json | https://satis.govcms.gov.au/develop |
-| `WHITELIST` | govcms-whitelist.json | https://satis.govcms.gov.au/whitelist |
+| `STABLE` | `govcms-stable.json` | https://satis.govcms.gov.au/ |
+| `MASTER` | `govcms-master.json` | https://satis.govcms.gov.au/master |
+| `DEVELOP` | `govcms-develop.json` | https://satis.govcms.gov.au/develop |
+| `WHITELIST` | `govcms-whitelist.json` | https://satis.govcms.gov.au/whitelist |
 
 They are updated by running `ahoy build` which populates the `./app` directory
 with static files.
 
-## When to update
+## Updating content
 
-Currently, updating is manual. The job to automate it was completed but it got stuck waiting
-for deployment tokens. In the meantime it has been found that manual intervention is usually needed.
+Updating of the Satis repository content is a manual process that includes 
```

Comment:
this is the only paragraph that was changed to explain what the process is.

The rest of the changes are formatting only.